cmf-cli/Handlers/PackageType/HtmlGulpPackageTypeHandler.cs

```diff
@@ -0,0 +1,128 @@
+﻿
+using System;
+using System.Collections.Generic;
+using Cmf.CLI.Builders;
+using Cmf.CLI.Commands.restore;
+using Cmf.CLI.Core.Enums;
+using Cmf.CLI.Core.Objects;
+using Cmf.CLI.Utilities;
+
+namespace Cmf.CLI.Handlers
+{
+    /// <summary>
+    /// Handler for UI packages
+    /// </summary>
+    /// <seealso cref="PresentationPackageTypeHandler" />
+    public class HtmlGulpPackageTypeHandler : PresentationPackageTypeHandler
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="HtmlGulpPackageTypeHandler" /> class.
+        /// </summary>
+        /// <param name="cmfPackage"></param>
+        public HtmlGulpPackageTypeHandler(CmfPackage cmfPackage) : base(cmfPackage)
+        {
+            List<Step> deploySteps; 
+            var minimumVersion = new Version("10.0.0");
+            // TODO: check if this is where this should be
+            // var projectConfig = FileSystemUtilities.ReadProjectConfig(this.fileSystem);
+            // var mesVersion = projectConfig.RootElement.GetProperty("MESVersion").GetString();
+            var mesVersion = "10.0.0"; 
+            var targetVersion = new Version(mesVersion!);
+            
+            if (targetVersion.CompareTo(minimumVersion) < 0)
```

Comment:
Shouldn't we undo this if? Seems like here we always do v9 and in HtmlNgCliPackageTypeHandler we always do v10